bucket/fvie.json

```
@@ -1,23 +1,29 @@
 {
-    "version": "1.5.0.0",
-    "description": "File VersionInfo Editor - A portable tool to edit VersionInfo metadata from Windows executables.",
-    "license": {
-        "identifier": "Freeware",
-        "url": "https://www.carifred.com/License.php"
-    },
-    "homepage": "https://www.carifred.com/fvie/",
-    "url": "https://www.carifred.com/fvie/FVIE.exe",
-    "hash": "cf4d0442b0a324358eb0b420c34a62ee41e1a39339e2842ed92cf5eee1aa9641",
-    "bin": "FVIE.exe",
-    "shortcuts": [
-        ["FVIE.exe", "FVIE"]
-    ],
-    "checkver": {
-        "url": "https://www.carifred.com/fvie/",
-        "regex": "Current version: \\n\\t\\t\\t([\\d\\.]+)"
-    },
-    "notes": [
-        "FVIE requires administrator privileges to run.",
-        "The program can edit the following files: .exe, .dll, .sys"
+  "version": "1.5.0.0",
+  "description": "File VersionInfo Editor - A portable tool to edit VersionInfo metadata from Windows executables.",
+  "license": {
+    "identifier": "Freeware",
+    "url": "https://www.carifred.com/License.php"
+  },
+  "homepage": "https://www.carifred.com/fvie/",
+  "url": "https://www.carifred.com/fvie/FVIE.exe",
+  "hash": "cf4d0442b0a324358eb0b420c34a62ee41e1a39339e2842ed92cf5eee1aa9641",
+  "bin": "FVIE.exe",
+  "shortcuts": [
+    [
+      "FVIE.exe",
+      "FVIE"
     ]
+  ],
+  "checkver": {
+    "url": "https://www.carifred.com/fvie/",
+    "regex": "Current version:\\s+([\\d.]+)"
+  },
+  "autoupdate": {
+    "url": "https://www.carifred.com/fvie/FVIE.exe"
+  },
+  "notes": [
+    "FVIE requires administrator privileges to run.",
+    "The program can edit the following files: .exe, .dll, .sys"
```

Completely useless note. If you want to keep it, add it into description.

Keep only priviledges

```suggestion
  "notes": "Adminitrator privileges are needed to execute FVIE"
```

---

I was hesitant whether to actually add that to the description as it could basically get too long.

But after seeing the manifest for `freeplane` I guess this isn't a problem.